README.md

```diff
@@ -181,6 +186,34 @@ This is the API where you provide configurations like runtime or something like
 
 If above block is configured in your pipeline then every time the spined pod will have a container named `node` which having the environments for nodejs8. By default pod will be spined with basic utilities like `oc`, `git` etc
 
+**Variables Configured in Pipeline**
+
+|         Name          |  Required  |  Default Value  |                    Description                      |
+|-----------------------|------------|-----------------|-----------------------------------------------------|
+|        runtime        |   false    |      none       |  runtime of the application ex java, node, go etc.  |
+|        version        |   false    |      none       |             version of the runtime using            |
+
+### plugins
+
+This is the API where you can provide the configuration related to the plugins. You can provide the configurations as map of key value pairs assigned to a plugin name
+
+```groovy
+    plugins analytics: ["disabled" : "true"]
+
+    plugins foobar: ["foo" : "bar"]
+
+```
+
+or
+
+```groovy
+    plugins analytics: ["disabled" : "true"], foobar: ["foo" : "bar"]
```

Comment:
shouldn't it be `true` and not `"true"`

Reply:
It will work that way also, should I update?